### ISSUE TITLE
cov2d

### ISSUE THREAD

**Issue by shanchao0906** [beifengchuizoudemeng/CWRU#1]
cov1d？

**Comment by beifengchuizoudemeng** [comment]
Conv2d
class torch.nn.Conv2d(in_channels,
	out_channels, kernel_size, stride=1, padding=0, dilation=1, groups=1,
	bias=True)
kernel_size (int or tuple) – Size of the convolving kernel
in my code:
self.conv = nn.Conv2d(in_channels=in_channels,out_channels=out_channels,kernel_size=(3,1),stride=1,padding=1)
kernel_size = (3,1) Equivalent to the size of the convolution kernel in one-dimensional convolution is set to 3

Conv1d
class torch.nn.Conv1d(in_channels, out_channels, kernel_size, stride=1, padding=0, dilation=1, groups=1, bias=True, padding_mode='zeros')
kernel_size (int or tuple) – Size of the convolving kernel ；The second dimension of the convolution kernel is determined by in_channels, so the size of the convolution kernel is actually kernel_size * in_channels.
Generally speaking, one-dimensional convolution nn.Conv1d is used for text data, only the width is convolved, and the height is not convolved.
